src/commands/ImportObjectsCommandGenerator.js

```diff
@@ -202,8 +207,14 @@ module.exports = class ImportObjectsCommandGenerator extends BaseCommandGenerato
 			validate: fieldValue => showValidationResults(fieldValue, validateArrayIsNotEmpty),
 		};
 		questions.push(questionListObjectsSelection);
+		return questions;
+	}
+
+	_generateQuestionsAfterObjectSelection(selectionObjectAnswers) {
+		const questions = [];
 
-		if (this._projectInfoService.getProjectType() === PROJECT_ACP) {
+		const customScript = selectionObjectAnswers.objects_selected.find(element => element.scriptId.startsWith(CUSTOM_SCRIPT_PREFIX));
+		if (this._projectInfoService.getProjectType() === PROJECT_ACP && customScript) {
```

Comment:
I think it will be cleaner to use:
```
const hasCustomScript = selectionObjectAnswers.objects_selected.some(element => element.scriptId.startsWith(CUSTOM_SCRIPT_PREFIX))
```
 because .some returns directly a boolean value if any of the elements in the array accomplishes the test function.

Reply:
Agree.